spec/samples/program_type/program_type_abridged_kitchen.json

```diff
@@ -1,9 +1,11 @@
 {
     "type": "ProgramTypeAbridged",
     "identifier": "2013::FullServiceRestaurant::Kitchen",
+    "display_name": "~2013::FullServiceRestaurant::Kitchen",
     "people": {
         "type": "PeopleAbridged",
         "identifier": "2013::FullServiceRestaurant::Kitchen_People",
+        "display_name": "//2013::FullServiceRestaurant::Kitchen_People",
```

Comment:
You might make some tests with Unicode characters, commas, or other unusual characteristics to test that serialization works correctly